### ISSUE TITLE
Quotes

### ISSUE THREAD

**Issue by anduong** [anduong/til#51]
"A common misconception about deep learning is that gradient descent is meant to reach the "global minimum" of the loss, while avoiding "local minima". In practice, a deep neural network that's anywhere close to the global minimum would be utterly useless (extremely overfit)" - François Chollet
